src/glmfit.jl

```diff
@@ -31,14 +31,18 @@ function GlmResp{V<:FPVector, D, L}(y::V, d::D, l::L, η::V, μ::V, off::V, wts:
     res
 end
 
-# returns the sum of the squared deviance residuals
+"""
+    deviance(r::GlmResp)
+
+Return the sum of the squared deviance residuals
```

Comment:
Actually, I don't think we should add a docstring for this method, as we already have one for `StatisticalModel` and for `LinearModel`. `GlmResp` sounds like an internal type to me, which should not be shown to the user when calling `?deviance`.